openvdb_points/tools/PointDataGrid.h

```diff
@@ -439,12 +471,13 @@ class PointDataLeafNode : public tree::LeafNode<T, Log2Dim> {
         return totalPointCount;
     }
 
-    void deactivateEmptyVoxels()
+    /// @brief Uses the point count of each voxel to update the active voxel mask on this
+    /// leaf node. Voxels with a point count of zero will be deactivated, and voxels with
+    /// a non-zero point count will be activated.
+    void updateActiveVoxelMask()
```

Comment:
updateActiveVoxelMask() is a bit of a mouthful and as there's no prior use in either codebase, I suggest we just go for updateActiveVoxels(). The alternative is two functions - activate() / deactivate(), but this might be a bit confusing as these two functions mean something subtly different for volumes.